app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_the_like_counter
+    post.update(likescounter: post.likescounter + 1)
+  end
```

Comment:
- [ ] Here we can also implement the rails active callback `after_save` method as mentioned in the post model